modules/admin-limit-operations.adoc

```diff
@@ -0,0 +1,80 @@
+// Module included in the following assemblies:
+//
+// ../scalability_and_performance/compute-resource-quotas.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="admin-limit-operations_{context}"]
+= Limit range operations
+
+== Creating a limit range
+
+To apply a limit range to a project, create the object, then create a limit range object definition with the required specifications.
```

Comment:
This sentence is somewhat confusing. `create the object`  for what? 
```suggestion
To apply a limit range to a project,  need to create a limit range object definition with the required specifications.
```

Reply:
This is done, thank you @wangke19 !